src/Xamarin.Android.Build.Tasks/Tasks/GetAdditionalResourcesFromAssemblies.cs

```diff
@@ -319,13 +323,13 @@ public override bool Execute ()
 					foreach (var ca in resolver.GetAssembly (assemblyItem.ItemSpec).CustomAttributes) {
 						switch (ca.AttributeType.FullName) {
 						case "Android.IncludeAndroidResourcesFromAttribute":
-							AddAttributeValue (androidResources, ca, "XA5206", "{0}. Android resource directory {1} doesn't exist.", true, fullPath);
+							AddAttributeValue (androidResources, ca, "XA5206", "{0}. (Details: Android resource directory {1} doesn't exist.)", true, fullPath);
```

Comment:
Thinking about it now, I realized that a quick way to check for other uses of parentheses in error messages was to look at the list in <https://github.com/xamarin/java.interop/blob/master/src/Java.Interop.Tools.Diagnostics/Java.Interop.Tools.Diagnostics/Diagnostic.cs>.  Indeed there are _not_ any other error messages that match this idea of using "(Details: ...)".

So maybe it would be best to split the specific file information into its own error line, if that wouldn't break convention?

Example:

> error XA5207: Unable to restore Java resources for package 'Foo'. The authors of the package should set the SourceUrl property or InstallInstructions property on the Java.Interop.JavaLibraryReferenceAttribute.
>
> error XA5207: Java library file foo.jar doesn't exist.